test/type.coffee

```diff
@@ -8,19 +8,33 @@ Amen.describe "Type functions", (context) ->
     isString, isFunction, isObject, isArray, isRegExp, isDate, isDefined,
     isGenerator, isPromise} = require "../src/type"
 
-  context.test "type"
-
-  context.test "isType"
-
-  context.test "instanceOf"
+  context.test "type", ->
+    assert type 7
```

Comment:
We should talk to Dan about `type` because I only have a partial understanding of its true purpose.